src/pages/config/ConfigList.tsx

```diff
@@ -111,7 +111,7 @@ export function ConfigListPage() {
               ...(configType
                 ? [
                     <BreadcrumbChild
-                      link={`/catalog?type=${configType}`}
+                      link={`/catalog?configType=${configType}`}
```

Comment:
Why are we changing this ?

Reply:
@moshloop  This is broken, they are supposed to be configType, renamed a while back because of the conflict with change type and other type as well, so config type is clearer.